### PR TITLE
EntityController::getAllPaginated search refactoring

### PR DESCRIPTION
**Pull request by cmsx**

### CODE REVIEW

File: Model/Model/IndexedModel.php [hunk 1]
```diff
@@ -128,18 +128,18 @@ public function getIndexDocumentData()
                     break;
                 } elseif ($results instanceof Collection) {
                     $nestedData = $results->map(function (IndexedModel $result) {
-                        return array_intersect_key($result->attributesToArray(), $result->mappingProperties);
+                        return array_intersect_key($result->attributesToArray(), $result->getMappingProperties());
```
Comment (jeremysik) [inline Model/Model/IndexedModel.php]: Where has `getMappingProperties()` been defined?

**Comment by jeremysik** [comment]
@cmsx looks good (just a question about `getMappingProperties()`. Need these tests to pass though, looks like something is wrong with the build.

File: Model/Model/IndexedModel.php (hunk 1 above)
Comment (cmsx) [inline Model/Model/IndexedModel.php]: @jeremysik `\Elasticquent\ElasticquentTrait::getMappingProperties` is just getter for that property

**Comment by cmsx** [comment]
@jeremysik I restarted travis tests and they finished successfully.

**Pull request merged**